docs/tempo/website/getting-started/tempo-in-grafana.md

```diff
@@ -0,0 +1,48 @@
+---
+title: Tempo in Grafana
+---
+
+[Grafana 7.4](https://grafana.com/grafana/download/7.4.5) and later have a built in Tempo datasource that can be used to query Tempo and visualize traces.  This page describes the high-level features and their availability.  It is recommended to use the latest versions for best compatibility and stability.
+
+1. [View trace by ID](#view-trace-by-id)
+2. [Log Search](#log-search)
+3. [Tempo Search](#tempo-search)
+4. [Service Map](#service-map)
+5. [View json file](#view-json-file)
+
+# View trace by ID
+The most basic functionality is to visualize a trace using its ID.  Select the Trace ID tab and enter the ID to view it. This functionality is enabled by default and is available in all versions of Grafana.
+<p align="center"><img src="../grafana-query.png" alt="View trace by ID"></p>
+
+# Log Search
+Traces can be discovered by searching logs for entries containing trace IDs.  This is most useful when your application also logs relevant information about the trace that can also be searched, such as HTTP status code, customer ID, etc.  This feature requires Grafana 7.5 or later, with a linked Loki data source, and a traceID derived field.
+
+<p align="center"><img src="../log-search.png" alt="Log Search"></p>
+
+
+# Tempo Search
+<span style="background-color:#f3f973;">This experimental feature is disabled by default. See below for more information on how to enable.</span>
+
+Tempo includes native search of recent traces.  Traces can be searched for data originating from a specific service, duration range, and span and process-level attributes included in your application's instrumentation, such as HTTP status code, customer ID, etc.  Currently only search of traces at the ingesters is supported. By default the ingesters store the last 15 minutes.
```

Comment:
> _By default the ingesters store the last 15 minutes._

I might have missed this, but in my understanding we store search data until the block gets flushed and how fast this happens depends on how much you ingest. Is _On average the ingester..._ more accurate?

Reply:
Nice, this works well for search